modules/related-posts/jetpack-related-posts.php

```diff
@@ -76,6 +76,9 @@ public function __construct( $blog_id_local, $blog_id_wpcom ) {
 		if ( ! class_exists( 'Jetpack_Media_Summary' ) ) {
 			jetpack_require_lib( 'class.media-summary' );
 		}
+
```

Comment:
Since the function `register_rest_field` is not in core but in WP-API, this should be wrapped in a conditional checking if the function exists to avoid breaking all REST API environment:
```php
// Add Related Posts to the REST API Post response.
if ( function_exists( 'register_rest_field' ) ) {
    add_action( 'rest_api_init',  array( $this, 'rest_register_related_posts' ) );
}
```
Otherwise, when WP-API is not installed or active, all endpoints (included ours like `wp-json/jetpack/v4/module/photon`) fail with a fatal error:
```
Fatal error: Call to undefined function register_rest_field() in /home/.../public_html/wp-content/plugins/jetpack/modules/related-posts/jetpack-related-posts.php on line 1307
```